tests/e2e/lib/dpa_helpers.go

```diff
@@ -120,6 +121,12 @@ func (v *DpaCustomResource) Build(backupRestoreType BackupRestoreType) error {
 		delete(defaultPlugins, oadpv1alpha1.DefaultPluginCSI)
 		delete(veleroFeatureFlags, "EnableCSI")
 		dpaInstance.Spec.SnapshotLocations = nil
+	case KOPIA:
```

Comment:
Can we merge this two cases and use your idea of `string(backupRestoreType)` for UploaderType?